### PR TITLE
Cleanse & backup page tidy

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/puxvINRI/453-possibly-document-cleanse-bug

This sorts out a minor task that was left hanging when the other cleanse work was done and also cleans up some unneeded whitespace while I was editing the file anyway.

### CODE REVIEW

File: omero/sysadmins/server-backup-and-restore.txt [hunk 1]
```diff
@@ -27,13 +23,15 @@ This can be performed daily using cron with a script such as:
     OMERO_PREFIX=/home/omero/OMERO-CURRENT
     $OMERO_PREFIX/bin/omero -s localhost -u $USERNAME -w $PASSWORD admin cleanse $BINARY_REPOSITORY
 
+Note that only items not listed in the SQL database (i.e. previously failed
```
Comment (jburel) [inline omero/sysadmins/server-backup-and-restore.txt]: I don't think we have used anywhere in the doc ``SQL database``.

File: omero/sysadmins/server-backup-and-restore.txt [hunk 2]
```diff
@@ -23,9 +23,9 @@ This can be performed daily using cron with a script such as::
     OMERO_PREFIX=/home/omero/OMERO-CURRENT
     $OMERO_PREFIX/bin/omero -s localhost -u $USERNAME -w $PASSWORD admin cleanse $BINARY_REPOSITORY
 
-Note that only items not listed in the SQL database (i.e. previously failed
-deletes) and now empty directories will be cleaned up by this script (see
-:ref:`DeleteBinaryData`).
+Note that only items not listed in the PostgreSQL database (i.e. previously
```
Comment (jburel) [inline omero/sysadmins/server-backup-and-restore.txt]: why not ``relational database``?

**Comment by hflynn** [comment]
Hopefully sorted now @jburel

**Comment by jburel** [comment]
@hflynn thanks looks good

**Pull request merged**